### ISSUE TITLE
No prompt for Corn, Corn, Corn

### ISSUE THREAD

**Issue by tfneptune** [throneteki/throneteki#3148]
Was playing a game and the challenge phase ended and the necessary conditions were in play to use Corn, Corn, Corn. There was no prompt at the end of the challenge phase to play the event.

**Comment by EvadLhorg** [comment]
Hey! thanks for reaching out and reporting this issue. Unfortunately, i wasn´t able to reproduce this. Are you sure there was a standing raven or steward character in play on your side at the end of the phase to pay the cost for the event? Do you by chance remember more of the specific situation or anything that could interact with it/make it not work?

**Comment by tfneptune** [comment]
Hey, sorry it took so long to respond. The first game I played was with my test partner. He did not do an int challenge plot 1 and I had a standing Winterfell Steward and there was no prompt to interrupt the end of the challenge for CCC. Again in the same game, plot 5, I had a standing 5 cost Aemon and CCC in hand. Aemon had a prompt to the end of the challenge but CCC did not.

We played another game to check out the scenario again to make sure we got it right. Again Aemon prompted but CCC did not. Here is a picture of that scenario.
![Capture](https://user-images.githubusercontent.com/22686516/125971936-3011133f-03e7-4068-94a0-939b341ec346.PNG)
